apps/rush-lib/src/cli/actions/InitAction.ts

```diff
@@ -0,0 +1,254 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as colors from 'colors';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+
+import { RushCommandLineParser } from '../RushCommandLineParser';
+import { BaseConfiglessRushAction } from './BaseRushAction';
+import { AlreadyReportedError } from '../../utilities/AlreadyReportedError';
+import { FileSystem, NewlineKind } from '@microsoft/node-core-library';
+import { CommandLineFlagParameter } from '@microsoft/ts-command-line';
+
+export class InitAction extends BaseConfiglessRushAction {
+  // Matches a well-formed BEGIN macro starting a multi-line section.
+  // Example:  /*[BEGIN "DEMO"]*/
+  //
+  // Group #1 is the indentation spaces before the macro
+  // Group #2 is the section name
```

Comment:
TSDoc?

Reply:
TSDoc is for public/protected contracts. These are just private notes.

---
In reply to: [217270696](https://github.com/Microsoft/web-build-tools/pull/823#discussion_r217270696) [](ancestors = 217270696)